tests/test_components_specs.py

```diff
@@ -0,0 +1,40 @@
+"""Express component specs test"""
+import os
+import pytest
+from express.exceptions import InvalidComponentSpec
+from express.component_spec import ExpressComponent
+
+valid_path = os.path.join("component_example", "valid_component")
+invalid_path = os.path.join("component_example", "invalid_component")
+
+
+def valid_express_schema() -> str:
+    return os.path.join(valid_path, "express_component.yaml")
+
+
+def valid_kubeflow_schema() -> str:
```

Comment:
this is not used ? I guess you want to add a test where you compare the generated one with this one ?

Reply:
indeed, was missing that one, just implemented one and will update